### ISSUE TITLE
Was xrv.baseline_models.chestx_det.PSPNet() trained on https://huggingface.co/datasets/BrachioLab/chestx ?

### ISSUE THREAD

**Issue by PabloMessina** [mlmed/torchxrayvision#158]
I was looking through the benchmarks to find any datasets with ground-truth segmentation masks of anatomical structures, but I couldn't find the one with classes like "Left Hilus Pulmonis" or "Facies Diaphragmatica". So I googled these keywords and came across https://huggingface.co/datasets/BrachioLab/chestx. Was this dataset the one used to train xrv.baseline_models.chestx_det.PSPNet() ? Can we access this dataset through torchxrayvision or do we need to download it on our own from Huggingface?

**Comment by ieee8023** [comment]
Here is the documentation: https://mlmed.org/torchxrayvision/models.html#torchxrayvision.baseline_models.chestx_det.PSPNet

It is this dataset: https://github.com/Deepwise-AILab/ChestX-Det-Dataset

It may be the same you are linking to, I'm not sure.

**Comment by ieee8023** [comment]
Hmm, I'm not sure. I reached out to the authors of the model to find out of the training data is available. Sorry I always thought the data was at that URL. It may not be made public.

I'm not sure if that link is the source of the data. Maybe try emailing one of the members of that lab.
